src/Nethermind/Nethermind.Db/MetadataDbKeys.cs

```diff
@@ -19,9 +19,9 @@ namespace Nethermind.Db
 {
     public static class MetadataDbKeys
     {
-        public static byte[] FinalizedBlockHash = { 1 };
+        public const int FinalizedBlockHash = 1;
```

Comment:
The most important thing is saving FinalizedBlockHash and FirstPoSBlock. I have a question mark about TerminalPoW.